client/extensions/woocommerce/style.scss

```diff
@@ -104,4 +103,110 @@
 		}
 	}
 
+	.products__product-form-variation-table {
+		border-spacing: 24px 16px;
+		border-collapse: collapse;
+
+		.products__product-form-variation-all-row {
+			background: lighten( $gray, 30% );
+		}
+
+		.products__product-dimensions-weight {
+			display: flex;
+			justify-content: flex-start;
+
+			.products__product-dimensions-input {
+				margin-right: 8px;
+
+				.products__product-dimensions-width,
+				.products__product-dimensions-height {
+					margin-left: -1px;
+				}
+
+				.products__product-dimensions-length,
```

Comment:
Not sure if it is a calypso rule but it is generally considered a bad idea to nest more than 3 deep in sass.